src/scss/includes/map_theme.scss

```diff
@@ -111,9 +111,10 @@
     background: rgba(255, 255, 255, 0.5);
   }
 
-  .map_control_group {
+  .mapboxgl-ctrl-group.map_control_group {
```

Comment:
This change seems to break the compass button on mobile:
![image](https://user-images.githubusercontent.com/4726554/48897641-e1ea0100-ee4a-11e8-9d44-dd9d90e1b3f8.png)